src/Coverage.jl

```diff
@@ -5,27 +5,20 @@
 # https://github.com/JuliaCI/Coverage.jl
 #######################################################################
 module Coverage
-
-    # The code coverage results produced by Julia itself report some
-    # lines as "null" (cannot be run), when they could have been run
-    # but were not (should be 0). We use JuliaParser to augment the
-    # coverage results by identifying this code.
-    # this file coverage will be checked by test/data/expected.info
```

Comment:
At first I was confused why this comment was removed, but apparently it only was moved down, into `amend_coverage_from_src!`, which makes sense.